integration-tests/basic-error/src/main/java/breaking/BreakingTest.java

```diff
@@ -0,0 +1,22 @@
+package breaking;
+
+public class BreakingTest {
+    public static void main(String[] args) {
+        System.out.println("In basic test");
+    }
+
+    private String neverUsed;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/sgoertzen/sonar-break-maven-plugin/file/2465882708/issues/source?bid=3070326&fileBranchId=3258144#l8)